docs/howtos/model_surgery.rst

```diff
@@ -98,7 +96,7 @@ After doing whatever you want, unflatten back:
 .. testcode::
 
   # Unflatten.
-  unflat_params = traverse_util.unflatten_dict({tuple(k.split('/')): v for k, v in flat_params.items()})
```

Comment:
I have seen this pattern a lot, that's why I decided to add the `sep="/"` parameter to make this a bit easier.

@marcvanzee added the original HOWTO, maybe he can elaborate on using tuples vs strings as keys of these dictionaries

Reply:
cc @marcvanzee ?

Reply:
This pattern of making "string/separated/keys" instead of tuples is so common in usage that I think I probably argued for the wrong format (tuples) when we did it this way.  So having a simple way to work w. stringy keys that's backwards compatible is quite nice and will help clean up a lot of common traversal code.